### PR TITLE
Add itmlogic

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by oblute** [comment]
@winstonjhoyle @Tata17 do you agree to maintain?

**Comment by oblute** [comment]
@conda-forge/staged-recipes this is ready for review

File: recipes/itmlogic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (ocefpaf) [inline recipes/itmlogic/meta.yaml:20]: I recommend you to add a Python lower bound here and on run. Even if this still builds on 2.7 you can use >=3

File: recipes/itmlogic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fa65eb9bff2a0ff3798bc10d6962874aa594d4ed9826d4a36c23cb3b81115e51
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```
Comment (ocefpaf) [inline recipes/itmlogic/meta.yaml:15]: Minor comment: if you use [grayskull](https://github.com/conda-incubator/grayskull) to create the recipes you will get the updated script line here. (And more nice things will be automatic populated for you.)

File: recipes/itmlogic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - python
+    - setuptools_scm
```
Comment (ocefpaf) [inline recipes/itmlogic/meta.yaml:24]: I'm pretty sure you don't need this on run. Can you try removing it in the feedstock later?

Comment (oblute) [reply recipes/itmlogic/meta.yaml:24]: You do need this on run. I added this in commit `394334f` which solved previous failures without it.

Comment (ocefpaf) [reply recipes/itmlogic/meta.yaml:24]: Interesting. I use setuptools_scm with the "write to file" option, so I never needed it on `run`.

Comment (xhochy) [reply recipes/itmlogic/meta.yaml:24]: Even without `write_to_file` you shouldn't need it. I briefly look at the source of `itmlogic` and cannot see the need. Feel free to ping me on the feedstock with a PR where the build fails and I can have a look.

`write_to_file` should write the version to an additional file but the version is already persisted in the package metadata in the case of an `sdist`. Additionally in an sdist, the VCS history is not available anymore, so `setuptools_scm` actually cannot do its magic anymore.

**Pull request merged**